src/benchmarks/matrix-multiplication.js

```diff
@@ -20,27 +20,27 @@
   var T1 = new Float32Array(16);
```

Comment:
As this benchmark is using load/store APIs, matrix-multiplication-load.js can be removed.